extensions/sql-migration/src/wizard/integrationRuntimePage.ts

```diff
@@ -661,7 +687,27 @@ export class IntergrationRuntimePage extends MigrationWizardPage {
 				// exit if new call has started
 				if (callSequence !== this._lastIn) { return; }
 
-				await utils.refreshAuthenticationKeyTable(this._view, this._authKeyTable, account, subscription, resourceGroup, location, migrationService);
+				// populate the table with data
+				await utils.refreshIntegrationRuntimeTable(this._view, this._integrationRuntimeTable, migrationServiceMonitoringStatus);
+
+				// based on the data, enable or disable configure ir button
+				if (migrationServiceMonitoringStatus.nodes.length === 4) {
+					this._configureIRButton.enabled = false;
+				}
+				else {
+					this._configureIRButton.enabled = true;
+				}
+
+				// if the versions are mismatched, show a warning
+				let nodeversion = migrationServiceMonitoringStatus.nodes[0].version;
```

Comment:
does it handle if migrationServiceMonitoringStatus.nodes is null or empty?

Reply:
Yes, it is handled.